docs/configuration.asciidoc

```diff
@@ -315,6 +315,24 @@ To reduce overhead and storage requirements, you can set the sample rate to a va
 between `0.0` and `1.0`. We still record overall time and the result for unsampled
 transactions, but no context information, tags, or spans.
 
+[float]
+[[config-server-cert]]
+=== `ELASTIC_APM_SERVER_CERT`
+
+[options="header"]
+|============
+| Environment                | Default
+| `ELASTIC_APM_SERVER_CERT`  |
+|============
+
+If you have configured your APM Server with a self signed TLS certificate, you can
+specify the path to the certificate via the `ELASTIC_APM_SERVER_CERT` configuration.
+This will enable the agent to verify the server's authenticity.
+
+By default, the agent verifies the server's certificate if you use an
+HTTPS connection to the APM server. Verification can be disabled by
+changing this setting to `false`.
```

Comment:
why can you set this to `false` ? Copy/paste error?

Reply:
copy pasta from the config below, thanks